src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java

```diff
@@ -49,7 +49,11 @@ public void actionPerformed(@NotNull AnActionEvent e) {
             NOTIFIER.notify(e.getProject(), file.getName() + " file has been registered with wskdeploy.", NotificationType.INFORMATION);
             EventUtils.publish(e.getProject(), ChooseWskDeployBinaryListener.TOPIC, l -> l.chooseWskDeployBinary(validWskDeploy.get()));
         } else {
-            NOTIFIER.notify(e.getProject(), file.getName() + " file cannot be used with wskdeploy.", NotificationType.ERROR);
+            if (file == null) {
+                NOTIFIER.notify(e.getProject(), "The file cannot be chosen", NotificationType.ERROR);
```

Comment:
In which case `file` becomes `null`?

Reply:
If the user doesn't select the file and close it, `file` will be `null`.

Reply:
I suppose in such a case, a user would just cancel the selection in most cases.
Do we really need to open a notification popup for it?

Reply:
Good point. Fixed the action to return if the user doesn't select the file.